scipy/stats/_continuous_distns.py

```diff
@@ -7637,6 +7682,8 @@ def _entropy(self, a, b):
 
     @extend_notes_in_docstring(rv_continuous, notes=fit_note)
     def fit(self, data, *args, **kwds):
+        if isinstance(data, CensoredData):
```

Comment:
I think this needs to be changed. (I might have done this last night.) I doesn't look necessary, and there needs to be a to path for fitting the censored data as it is.